### DISCUSSION TITLE
What input devices does Bevy support?

### DISCUSSION THREAD

**Discussion by vladislav1nikov** [bevyengine/bevy#8166] [Q&A] [answered]
Keyboard, mouse and touch input, what else? What about gamepads and TV remotes?

**Comment by Nilirad** [comment, marked as answer]
From the API documentation, the supported input methods should be these:

- gamepad
- keyboard
- mouse
- touch

Source: https://docs.rs/bevy_input/0.10.0/bevy_input/
